resources/production/config-example.toml

```diff
@@ -7,7 +7,7 @@
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
 
 # Whether to synchronize all data back to genesis, or just [deploys.max_ttl] (chainspec setting) worth of data.
-sync_to_genesis = true
+sync_handling = 'ttl'
```

Comment:
It would be good to copy the comment block from `resources/local/config.toml` as it's more likely that will find its way into the operators' config.toml eventually I expect.

Reply:
good idea; done